ontology/nomen.md

```diff
@@ -40,5 +40,3 @@ usages:
   user: https://taxonworks.org
 activity_status: active
 ---
-
-NOMEN is a nomenclatural ontology for biological names (not concepts).  See the <a href="https://github.com/SpeciesFileGroup/nomen">NOMEN homepage</a> for more info.
```

Comment:
The homepage link is redundant of the structured metadata, then the remaining sentence doubled the short description, so this is removed.